packages/rest-specs/README.md

```diff
@@ -0,0 +1,36 @@
+# Cadl Ranch Specs
+
+This package contains all the scenarios that should be supported by a client generator.
+
+## Development
```

Comment:
think those instruction need to be updated/cleaned up

Reply:
Marked as pending